### PR TITLE
[EWL-5041] SG2 | Create "JAMA as 3 Up" variant

### PR DESCRIPTION
**Pull request by csj1012**
## Ticket(s)

**Jira Ticket**
- [EWL-5041: SG2 | Create "JAMA as 3 Up" variant](https://issues.ama-assn.org/browse/EWL-5041)

## Description
- Adds a "JAMA as Three Up" variant of the "JAMA" molecule.
- Adds the molecule to the "News with JAMA Examples" page.

## To Test
- [ ] `gulp serve`
- [ ] Given that I am viewing SG2
- [ ] When I click "Pages" in the menu
- [ ] Then I should see "News with JAMA Examples"
- [ ] When I click "News with JAMA Examples" and scroll to the bottom of the page (look above the four-up teaser)
- [ ] Then I should see the "Logo as JAMA" atom
- [ ] And I should see a red horizontal rule
- [ ] And I should see 3 article images
- [ ] And I should see 3 article titles (linked)
- [ ] When I hover over the article titles
- [ ] Then they should turn red and underlined
- [ ] And I should see a gray horizontal rule
- [ ] And I should see the link "Sign-up for a membership to the AMA and receive a subscription to JAMA"
- [ ] And the link should be red

## Visual Regressions

- [x] **Before proceeding:** Run visual regression tests locally to ensure new work does not introduce new bugs.

- [x] _If the new work does your work introduce visual regressions into existing work in the Style Guide (or in a Drupal 8 environment)_, either call these out here or address them before marking the PR as `ready for review`.

- [x] If you are creating or updating a pattern be sure you have created or updated the [scenario in `backstop.json`](ama-style-guide-2/styleguide/backstop.json) and have included a new or updated screenshot in [`bitmaps_reference`](ama-style-guide-2/styleguide/backstop_data/bitmaps_reference).

## Relevant Screenshots/GIFs
screen shot 2018-05-18 at 12 44 57 pm

## Remaining Tasks
N/A

## Additional Notes
- There is a preexisting visual regression happening in the two-column right layout. @labbydev has created https://issues.ama-assn.org/browse/EWL-5115 to address it. 
- The "Three up" and "Content bottom" variants of the "JAMA" molecule intentionally do not show the proper styling at the molecule level. See pattern info for those patterns. Per convo with @labbydev I am updating the ticket acceptance criteria accordingly.

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/_patterns/05-pages/news-as-gated.twig [hunk 1]
```diff
@@ -17,8 +17,10 @@
   {% set promo = promoAsInline %}
   {% include '@molecules/promo.twig' %}
   {% include '@atoms/paragraph.twig' %}
-  {% include '@molecules/tags.twig' %}
-  {% include '@organisms/four-up-teaser.twig' %}
+  <section class="ama__page--news__teasers">
```
Comment (labbydev) [inline styleguide/source/_patterns/05-pages/news-as-gated.twig:20]: Can we label this something more generic and less tied to what is meant to be in this section. I suspect what is meant to be in this section may change over time. I suggest referring to it as "content bottom" or something else that refers more to it's location on the page than it's content.

File: styleguide/source/_patterns/05-pages/news-with-jama-examples.twig [hunk 2]
```diff
@@ -18,7 +18,10 @@
   {% include '@atoms/paragraph.twig' %}
   {% include '@molecules/jama.twig' with {jama : jamaAsContentBottom} %}
   {% include '@molecules/tags.twig' %}
-  {% include '@organisms/four-up-teaser.twig' %}
+  <section class="ama__page--news__teasers">
```
Comment (labbydev) [inline styleguide/source/_patterns/05-pages/news-with-jama-examples.twig:21]: See same comment on the "News as Gated" page

**Review by labbydev — COMMENTED** [review]
I have thoughts. Feel free to argue against them.

File: styleguide/source/_patterns/05-pages/news-as-gated.twig (hunk 1 above)
Comment (csj1012) [reply styleguide/source/_patterns/05-pages/news-as-gated.twig:20]: @labbydev I agree, but we're already calling the variant that appears at the bottom of the body content "Content Bottom" - do you have any other ideas?

Comment (labbydev) [reply styleguide/source/_patterns/05-pages/news-as-gated.twig:20]: 👍

**Pull request merged**